static/css/app.css

```diff
@@ -185,15 +185,26 @@ h1, h2, h3, h4, h5, h6 {
   outline: 0;
 }
 
-#menu-button.hidden {
+#menu-button {
+  background-image: url('../images/menu.png');
+}
+
+#back-button {
+  background-image: url('../images/back.png');
+}
+
+#menu-button.hidden,
+#back-button.hidden {
   display: none;
 }
 
-#menu-button::-moz-focus-inner {
+#menu-button::-moz-focus-inner,
+#back-button::-moz-focus-inner {
   border: 0;
 }
 
 #menu-button:hover,
+#back-button:hover,
```

Comment:
Nit: This doesn't seem to be working for some reason.

Reply:
Fixed. There was a trailing comma further down.